### DISCUSSION TITLE
Exit2Studios Pool (Austin TX)

### DISCUSSION THREAD

**Discussion by Exit2Studios** [tagyoureit/nodejs-poolController#520] [My Pool]
We are located in Austin, Texas and built our pool in 2009 .  The actual build was quite the process, and 13 years later I continue to have a love/hate relationship with my pool builder, who provides no warranty at this point (or frankly ever), but does have a steel trap memory that I dig into on occasion when things go strange.

![Hodel+Jul21_900](https://user-images.githubusercontent.com/16908875/178113140-6b246611-34d6-47db-b260-436fe679e4f3.png)

Our pool is unique in that the spa is located in the infinity edge catch basin.  This has been a great option for us, but presents some challenges/dangers compared to normal pools where the spa empties into the actual pool body.  That combined with the fact that our pool equipment pad is significantly higher than our pool, leads to some difficulty in priming pumps, etc.

<img width="605" alt="image" src="https://user-images.githubusercontent.com/16908875/178113695-617d1e9d-d9d1-45e8-bf62-b94759b539a9.png">

- Approximately 25,000 gallons (water meter stopped during initial fill --- yay!)
- Diamond Brite "Midnight Blue" Plaster (not recommended -- beautiful but has not held up well)
- Glass tile negative Edge spa (glass tile has been a real challenge despite the beauty -- tread with caution if you choose it)
- Pentair VSP main Pump (replaced original 2-speed Whisperflo)
- Pentair Whisperflo Single Speed Feature Pump
- Pentair FNS plus 60 DE filter
- Pentair MasterTemp Gas Heater
- Pentair Warrior SE Robotic Cleaner 
- Stenner Chlorine Pump
- Stenner MA Pump (not hooked up yet)
- Flow Cell (not hooked up yet)

![PoolPadPic](https://user-images.githubusercontent.com/16908875/178114525-eb146443-838f-449c-ade7-f24c236efa84.jpg)

My Pool is currently controlled by the Hayward/Goldline that was installed by our Pool Builder.  I am in the process of testing my njsPC setup in my "lab" prior to deploying to the pool.  I have previously controlled the Intelliflo with njsPC, but am not at the moment, as I need the ability to control it from the pad from time to time.

**Comment by rstrouse** [comment]
Beautiful pool!  We did just add the ability to enter service modes using buttons on the box so maybe that will help you a bit.  Interesting that the spa is in the catch basin.

**Reply by tagyoureit** [reply]
Agree it's an amazing looking pool!
